### DISCUSSION TITLE
aggregationType support

### DISCUSSION THREAD

**Discussion by vmalisz** [joshcarty/google-searchconsole#62] [Ideas]
Without the aggregationType parameter ability, the data by URL only sums the metrics from the query report which is wrong. Would it be possible to add that functionality?

https://developers.google.com/webmaster-tools/v1/searchanalytics/query

thank you for your amazing work, Google's API doc is so unclear... you made my life easier

EDIT: it might not be necessary. After running some tests it looks like the limitation comes from the API, and the aggregation parameter is pretty much useless. If one wants the total clicks, and impressions by page, just remove all other dimensions from the query. Adding "aggregationType = byPage" won't give you the total otherwise. 
I guess it makes sense, they are not able to detect every device for instance.

**Comment by joshcarty** [comment]
Thanks for following up on this @vmalisz. Having a cleanup and going to remove this but feel free create an [Issue](https://github.com/joshcarty/google-searchconsole/issues) if you do think this is a bug or missing feature.
